src/workflows-app/components/ImportGithub.test.js

```diff
@@ -198,4 +205,45 @@ describe('Add a Workflow Link', () => {
     // ** ASSERT **
     expect(postMethodFunction).toHaveBeenCalledTimes(0);
   });
+
+  it('shows modal on successful import', async () => {
+    isFeaturePreviewEnabled.mockImplementation((id) => (id === ENABLE_WORKFLOWS_SUBMISSION_UX_REVAMP ? true : isFeaturePreviewEnabled(id)));
+
+    const githubLink = 'https://github.com/broadinstitute/cromwell/blob/develop/wdl/transforms/draft3/src/test/cases/simple_task.wdl';
+    const postMethodFunction = jest.fn(() => Promise.resolve({ method_id: 'abc123' }));
+    const user = userEvent.setup();
+
+    await Ajax.mockImplementation(() => {
+      return {
+        Cbas: {
+          methods: {
+            post: jest.fn(postMethodFunction),
+          },
+        },
+      };
+    });
+
+    workflowsAppStore.set({
+      workspaceId: '79201ea6-519a-4077-a9a4-75b2a7c4cdeb',
+      cbasProxyUrlState: { status: AppProxyUrlStatus.Ready, state: 'https://lz-abc/terra-app-abc/cbas' },
+    });
+
+    // ** ACT **
+    render(h(ImportGithub, { setLoading: jest.fn(), signal: jest.fn(), onDismiss: jest.fn(), workspace }));
+
+    const urlLink = screen.getByPlaceholderText('Paste Github link');
+    const workflowName = screen.getByPlaceholderText('Workflow Name');
+
+    await user.type(urlLink, githubLink);
```

Comment:
I think we're actually supposed to go back to using fireEvent for typing: https://broadinstitute.slack.com/archives/C01EHNUM73R/p1693514267660619

Reply:
Looking at this doc, do we think this is only for `.ts` files? I see that there's another link to a javascript best practices doc and it mentions using user as the above, but no mention of `fireEvent`.